pylint/checkers/variables.py

```diff
@@ -2673,18 +2672,18 @@ def _check_unpacking(
                     "unbalanced-tuple-unpacking",
                     node=node,
                     args=(
-                        _get_unpacking_extra_info(node, inferred),
+                        details,
                         len(targets),
+                        "s" if len(targets) > 1 else "",
```

Comment:
```suggestion
                        "" if len(targets) == 1 else "s",
```

Reply:
Damn TIL ! a search for "0 values or 0 value" bring nothing but it seems that apple is the universal example. https://www.quora.com/Is-zero-plural-You-have-one-apple-I-have-zero-appleS

Reply:
😁🍎